### PR TITLE
Updated and corrected Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

File: locales/nl.json [hunk 1]
```diff
@@ -1,315 +1,315 @@
...
+    "Rating: ": "Waardering",
+    "Language: ": "Taal",
+    "View as playlist": "Tonen als afspeellijst",
+    "Default": "Standaard",
+    "Music": "Muziek",
+    "Gaming": "Gaming",
+    "News": "Nieuws",
+    "Movies": "Films",
+    "Download": "Downloaden",
+    "Download as: ": "Downloaden als",
```
Comment (omarroth) [inline locales/nl.json]: Should this be `Downloaden als: `, or is there a reason we leave out the colon here?

File: locales/nl.json [hunk 2]
```diff
@@ -1,315 +1,315 @@
...
-    "`x` subscriptions": "`x` abonnees",
-    "`x` tokens": "",
-    "Import/export": "Importeer/Exporteer",
-    "unsubscribe": "abonnement opzeggen",
-    "revoke": "",
-    "Subscriptions": "Abonnees",
-    "`x` unseen notifications": "`x` onbekeken notificaties",
+    "Administrator preferences": "Beheerdersinstellingen",
+    "Default homepage: ": "Standaard startpagina: ",
+    "Feed menu: ": "Feedmenu",
```
Comment (omarroth) [inline locales/nl.json]: Similar question here, should this be `Feedmenu: `?

File: locales/nl.json (hunk 1 above)
Comment (Vistaus) [reply locales/nl.json]: Oops, you're right: I accidentally missed the colon there.

File: locales/nl.json (hunk 2 above)
Comment (Vistaus) [reply locales/nl.json]: Also missed the colon here, sorry about that!

**Comment by Vistaus** [comment]
Fixed :)

File: locales/nl.json [hunk 3]
```diff
@@ -1,315 +1,315 @@
...
+    "Top enabled? ": "Bovenkant inschakelen",
+    "CAPTCHA enabled? ": "CAPTCHA gebruiken?",
+    "Login enabled? ": "Inloggen toestaan?",
+    "Registration enabled? ": "Registratie toestaan?",
+    "Report statistics? ": "Statistieken bijhouden?",
```
Comment (omarroth) [inline locales/nl.json]: Likely a couple elsewhere I missed, but these 4 appear to be missing the trailing space, should be `Statistieken bijhouden? `, `Registratie toestaan? `, etc.

Comment (Vistaus) [reply locales/nl.json]: You're right, I did not expect spaces there, so I missed them. Will fix that too.

Comment (Vistaus) [reply locales/nl.json]: Also fixed now :)

**Comment by omarroth** [comment]
LGTM :+1:

**Pull request merged**